src/index.ts

```diff
@@ -461,7 +489,7 @@ export async function init(config: IClientConfig): Promise<EppoClient> {
       }
     }
 
-    if (!initializationSource) {
+    if (initializationSource === 'did not finish') {
```

Comment:
This is more clear good thinking. Given how much we use these in the code repeatedly, perhaps we should make it an enum or something.